user_manual/ios_faq.rst

```diff
@@ -0,0 +1,15 @@
+===========
+iOS App FAQ
+===========
+
+**Issue:**
+
+Enabling the instant upload feature requires that the Owncloud client have access to location services. 
+
+**Resolution:**
+
+iOS does not allow any execution in background. The ownCloud iOS App use the location to wake up the app in background in order to check if there is any new image to be uploaded.
```

Comment:
@carlaschroder this is not completely like this "iOS does not allow any execution in background" the iOS app does execution in background, uploads and downloads if they were in progress, certainly, what it doesn't do is track the location in the background.
Having said that, it'd be worthy to wait to merge this, since the instant uplaod behavior is being updated as part of: https://github.com/owncloud/ios/pull/681